content/help/handle-split-sites.md

```diff
@@ -0,0 +1,32 @@
+# "Split" Sites: Connect Your Smart Inverter for Solar Energy Forecasting
+
+If you have a "split" site with multiple sets of solar panels pointing in different directions or tilted at different angles, there are two methods you can use to enter site details for solar energy forecasting:
+
+## Method 1: Create a Separate Site for Each Direction/Tilt
+
+The first method involves creating a new site for each differing direction or tilt. This method is useful if you want to track the performance of each set of solar panels separately. To use this method, follow these steps:
+
+1. Add a new site for each set of solar panels with a different direction or tilt.
+2. Enter the direction and tilt for each site accurately.
+3. Enter the module capacity and inverter capacity for each site.
+
+This method may result in more work for you, as you'll need to track the performance of each site separately.
+
+## Method 2: Connect Your Smart Inverter
```

Comment:
put this method first, because this is the preferred method

Reply:
I think it's fine to keep it second—it's definitely the less accessible option